go.mod

```diff
@@ -23,7 +23,8 @@ require (
 	github.com/containerd/console v1.0.3
 	github.com/containerd/containerd v1.6.8
 	github.com/go-acme/lego/v4 v4.9.0
-	github.com/google/go-containerregistry v0.12.0
```

Comment:
downgrading this to v0.11.0 is a problem. what did you do to trigger this?

Reply:
unsure I think maybe on one of the rebases it snuck in there, reverted and tested and it looks good now